cpp/src/interop/to_arrow.cu

```diff
@@ -15,10 +15,14 @@
  */
 
 #include <cudf/column/column.hpp>
+#include <cudf/column/column_factories.hpp>
 #include <cudf/column/column_view.hpp>
+#include <cudf/copying.hpp>
 #include <cudf/detail/interop.hpp>
 #include <cudf/detail/iterator.cuh>
+#include <cudf/detail/null_mask.hpp>
 #include <cudf/detail/nvtx/ranges.hpp>
+#include <cudf/detail/scatter.hpp>
```

Comment:
```suggestion
```
I don't think this is used here. Perhaps recheck if all of these are still needed?

Also, I noticed a `cudf::copy_bitmask()` call on line 84 that should probably be `cudf::detail::copy_bitmask` and pass the `stream` to it. Maybe it can be fixed with this PR?

Reply:
Good eye. From a cursory look I don't think the others can be removed but I'll test them just in case.

Reply:
I fixed copy_bitmask usage, which also let me drop the null_mask header (still need detail/null_mask but that was already being included).